src/endpoints/v1.es6.js

```diff
@@ -217,14 +217,19 @@ class APIv1Endpoint {
     cache.set(key, data);
   }
 
-  get subreddits () {
+  get subreddits() {
     return bind({
-      buildOptions: function(options) {
-        var uri = this.origin + `/r/${options.query.subreddit}/about.json`;
+      buildOptions: function (options) {
+        var uri = this.origin;
+        if (options.query.where) {
+          uri += `/subreddits/${options.query.where}.json`;
```

Comment:
It's not `sort` it's more likely `filter` or `type`... I can do `sort`, I don't care.